sql/core/src/main/scala/org/apache/spark/sql/execution/command/commands.scala

```diff
@@ -237,15 +237,18 @@ case class ExplainCommand(
     logicalPlan: LogicalPlan,
     override val output: Seq[Attribute] =
       Seq(AttributeReference("plan", StringType, nullable = true)()),
-    extended: Boolean = false)
+    extended: Boolean = false,
+    codegen: Boolean = false)
```

Comment:
i like the sql -- just saying it'd be great to have different command classes rather than having different flags in a single command.